source/analysis/AMNormalizationAB.h

```diff
@@ -0,0 +1,100 @@
+#ifndef AMNORMALIZATIONAB_H
+#define AMNORMALIZATIONAB_H
+
+#include "analysis/AMStandardAnalysisBlock.h"
+
+/// This analysis block accepts any number of identically ranked data sources and divides one by the other to normalize one designated as "data".
```

Comment:
Is the 'any number' part of this comment correct? It looks to me like you have a single data source to analyse and the normalizing data source.